src/types/constants.ts

```diff
@@ -0,0 +1,26 @@
+export enum ISSUE_LABELS {
+  // Profit labels
+  PROFIT_LT_10 = "Profit: <10%",
+  PROFIT_LT_50 = "Profit: <50%",
+  PROFIT_LT_100 = "Profit: <100%",
+  PROFIT_GTE_100 = "Profit: 100%+",
+
+  // Time labels
+  TIME_LT_1DAY = "Time: <1 Day",
+  TIME_LT_1WEEK = "Time: <1 Week",
+  TIME_LT_2WEEKS = "Time: <2 Weeks",
+  TIME_LT_1MONTH = "Time: <1 Month",
+  TIME_GTE_1MONTH = "Time: 1 Month+",
+
+  // Price Labels
+  PRICE_100_1000 = "Price: 100-1000 USDC",
+  PRICE_200_2000 = "Price: 200-2000 USDC",
+  PRICE_300_3000 = "Price: 300-3000 USDC",
+  PRICE_400_4000 = "Price: 400-4000 USDC",
+  PRICE_500_5000 = "Price: 500-5000 USDC",
+
+  PRICE_100_400 = "Price: 100-400 USDC",
+  PRICE_500_2000 = "Price: 500-2000 USDC",
+  PRICE_1000_4000 = "Price: 1000-4000 USDC",
+  PRICE_2000_8000 = "Price: 2000-8000 USDC",
+}
```

Comment:
We should allow our customers to configure these. Something to bear in mind soon! From what I understand, GitHub Apps (bots) are normally configured with a .yml file in customer repositories. Perhaps you know the best way to proceed here.

Reply:
Yeah that's a good catch and what I was thinking during the research. The blocker is that we don't have any standard names/interfaces to make it configurable. We still have some unclear things as the followings.

Their time labels would be different from ours, 
Their price labels would be different from ours, 
Their price calculation logic would be different from ours.
...

From above unclear things, I am just focusing on ubiquity-specific bot on this pull request. 
we will make it configurable step by step.

Reply:
For whatever its worth, I somewhat expect early partners to just copy our lead in order to onboard more easily. We spent a fair amount of time prototyping and using these values and they seem to be mostly good. I just think that the base rate is a simple but major variable. Perhaps just making base rate configurable and providing a few examples of how the entire pricing chart will look if modifying the base rate will be sufficient for getting our early partners to start with their own price ranges that make sense to them?

Reply:
Hmm, I can make it configurable as of now.

Reply:
https://github.com/ubiquity/bounty-bot/pull/32/commits/d244ae24e2e800519fa972b538acae1bed610a60